### PR TITLE
[fix] Fix scroll on newer versions of Chrome (>=61) + remove unused code

### PR DESCRIPTION
**Pull request by lpagliari**
Newer versions of Chrome use the same way of Firefox to handle scrolling.

Also remove some trailing spaces.

### CODE REVIEW

File: css.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-exports.aceEditorCSS = function(hook_name, cb){return ["/ep_cursortrace/static/css/cursortrace.css"];} // inner pad CSS
```
Comment (lpagliari) [inline css.js]: This is not used anywhere. The hook used is on `static/js/css.js`

File: handleMessage.js [hunk 2]
```diff
@@ -64,11 +64,7 @@ exports.handleMessage = function(hook_name, context, callback){
     });
   }
 
-  if(iscursorMessage === true){
-    callback([null]);
-  }else{
-    callback(true);
-  }
```
Comment (lpagliari) [inline handleMessage.js]: Line 32 already checks for `iscursorMessage`, and returns if it enters the conditional:

```
   if(!iscursorMessage){	   
     callback(false);	     
     return false;	     
   }
```

Code on `else` is never reached.

File: static/js/main.js [hunk 3]
```diff
@@ -64,14 +63,6 @@ exports.className2Author = function(className)
   return null;
 }
 
-var lineAndColumnFromChar = function(x)
```
Comment (lpagliari) [inline static/js/main.js]: function not used anywhere

File: static/js/main.js [hunk 4]
```diff
@@ -149,31 +139,31 @@ exports.handleClientMessage_CUSTOM = function(hook, context, cb){
...
 
-      // A load of fugly HTML that can prolly be moved ot CSS
+      // A load of ugly HTML that can prolly be moved to CSS
       var newLine = "<span style='width:"+divWidth+"px' id='" + authorWorker + "' class='ghettoCursorXPos'>"+newText+"</span>";
 
       // Set the globalKey to 0, we use this when we wrap the objects in a datakey
       globalKey = 0; // It's bad, messy, don't ever develop like this.
 
       // Add the HTML to the DOM
-      var worker = $('iframe[name="ace_outer"]').contents().find('#outerdocbody').append(newLine);
```
Comment (lpagliari) [inline static/js/main.js]: `var worker` is overwritten on the next line

File: static/js/main.js [hunk 5]
```diff
@@ -232,16 +222,13 @@ exports.handleClientMessage_CUSTOM = function(hook, context, cb){
...
           // Remove all divs that already exist for this author
           $('iframe[name="ace_outer"]').contents().find(".caret-"+authorClass).remove();
-  
+
           // Location of stick direction IE up or down
           if(stickUp){var location = 'stickUp';}else{var location = 'stickDown';}
-  
-          // Location of stick direction IE up or down
-          if(stickLeft){var locationLR = 'stickLeft';}else{var locationLR = 'stickRight';}
-
```
Comment (lpagliari) [inline static/js/main.js]: `locationLR` is not used anywhere

File: static/js/main.js [hunk 6]
```diff
@@ -255,42 +242,28 @@ exports.handleClientMessage_CUSTOM = function(hook, context, cb){
...
-/***
- * 
- *  Once ace is initialized, we bind the functions to the context
- * 
- ***/
-
-exports.aceInitialized = function(hook, context){
-  var editorInfo = context.editorInfo;
-  padEditor = context.editorInfo.editor;
-}
```
Comment (lpagliari) [inline static/js/main.js]: this hook only set `padEditor`, and `padEditor` is not used anywhere

File: static/js/main.js [hunk 7]
```diff
@@ -324,12 +297,10 @@ function html_substr( str, count ) {
   return div.innerHTML;
 }
 
-function wrap(target, key) { // key can probably be removed here..
```
Comment (lpagliari) [inline static/js/main.js]: removed `key` from function

**Comment by JohnMcLear** [comment]
Good to hear are still alive man!  Hi from me and Peter


________________________________

**Comment by redhog** [comment]
Heyas! Sure I'm alive :) Say hi to Peter! We should meet up again some time! I'm however living near the north pole nowdays - https://www.google.com/maps/place/Troms%C3%B8/@69.6652941,18.8477109,12z/data=!3m1!4b1!4m5!3m4!1s0x45c4c4526c3b71fd:0x23dca858e6ebed3!8m2!3d69.6492047!4d18.9553238

**Comment by JohnMcLear** [comment]
Bump me through an email, john dot mclear dot co dot uk and we can organize a meet!




Secret, Proprietary & Confidential

This electronic message transmission contains information which is proprietary and confidential. The information is intended solely for the intended recipient, and use by any other party is not authorized.
________________________________

**Pull request merged**